src/app/app.component.ts

```diff
@@ -51,6 +51,7 @@ export class MyApp {
       let isIE: boolean = navigator.userAgent.indexOf('Trident', 0) !== -1;
       this.infoSvc.setInternetExplorer(isIE);
       this.runningInBrowser = this.platform.is('mobileweb') || this.platform.is('core');
+      this.notOnDesktop = !this.platform.is('core');
```

Comment:
Well, wait. I did this so that any platform other than desktop will have the menu close button. Would you prefer that I call it `onDesktop` and write `<button ion-button ... *ngIf="!onDesktop">`? For reference, [this is where I used `notOnDesktop`](https://github.com/umts/pvta-multiplatform/pull/404/commits/b6457dc9a91eeba91c6ea62f953a49221e658a2c)